internal/future/serving/testing/server_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testing
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/open-match/internal/future/serving"
+	"github.com/GoogleCloudPlatform/open-match/internal/pb"
+
+	shellTesting "github.com/GoogleCloudPlatform/open-match/internal/future/testing"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+)
+
+// TestMustParamsForTesting verifies that a server can stand up in insecure mode.
```

Comment:
Do these tests add more coverage than the ones already testing server / insecure / tls within the serving package?

Reply:
It just verifies that the servers come up with the test methods instead of manual wireup.